api/descriptions/policies/compliance_container_get.md

```diff
@@ -1,9 +1,18 @@
-Retrieves a list of all rules that make up your compliance policy for containers.
+Retrieves the compliance policy for containers protected by Defender.
+A policy consists of ordered rules.
+
+This endpoint maps to the policy table in **Defend > Compliance > Containers and images > Deployed** in the Console UI.
+
+### cURL Request
+
+The following cURL command retrieves all rules in the policy.
 
 ```bash
 $ curl -k \
   -u <USER> \
   -H 'Content-Type: application/json' \
   -X GET \
-  https://<CONSOLE>:8083/api/v1/policies/compliance/container
+  'https://<CONSOLE>/api/v1/policies/compliance/container?project=<PROJECT_NAME>'
```

Comment:
@KICKdesigns Let's remove the `project` query param from all curl commands to reduce the number of `<VARS>` customers need to fill into to run a working a curl command. I'll fix this for you so we can merge this now.

Reply:
@KICKdesigns Fixed